src/Monitor/Monitor.Test/Monitor.Test.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Monitor" Version="0.25.2-preview" />
```

Comment:
Please also update the Monitor sdk version in other modules' test project. Or the their tests will fail, which block the PR.

Reply:
Yes, the latest it's 0.25.3-preview